### ISSUE TITLE
Air-light 7.5.0 fails Theme Check for the official WordPress theme directory

### ISSUE THREAD

**Issue by ronilaukkarinen** [digitoimistodude/air-light#110]
**One or more errors were found for Air-light.**

WARNING: file_get_contents was found in the file inc/hooks/acf-blocks.php File operations should use the WP_Filesystem methods instead of direct PHP filesystem calls.Line 48: $block['icon'] = file_get_contents( $icon_path );

**Comment by Zhoha28** [comment]
Found 2 more errors when done theme check on fresh install-

```
REQUIRED: The theme uses the register_taxonomy() function in the file inc/includes/taxonomy.php. register_taxonomy() is plugin-territory functionality and must not be used in themes. Use a plugin instead.
Line 61: register_taxonomy( $slug, $object_types_slugs, $args );
Line 64: // custom taxonomies for custom post types. Use register_taxonomy_for_object_type()
Line 69: $register_result[ $object_type ] = register_taxonomy_for_object_type( $slug, $object_type );

REQUIRED: The theme uses the register_post_type() function in the file inc/includes/post-type.php. register_post_type() is plugin-territory functionality and must not be used in themes. Use a plugin instead.
Line 48: * @see http://codex.wordpress.org/Function_Reference/register_post_type
Line 58: return register_post_type( $slug, $args );
```

**Comment by ronilaukkarinen** [comment]
@Zhoha28 Couple of problems in your comment:

1. Why did you comment on this old, closed issue? Did you use version 7.5.0? Always open a separate issue if the comment is not related to the issue at hand.

2. Did you check against the theme version you **downloaded from GitHub** or did you check after installing from the **[official theme directory](https://wordpress.org/themes/air-light/)**? Because if you did install from GitHub and check that, it should fail, because it's a dev theme and not official one. As you can see in [trac](https://themes.trac.wordpress.org/ticket/126967), latest version passes all the tests. Otherwise it would not be there at all, logic, right?

It seems you used the GitHub version that indeed has those files, on purpose. However, these files are not present in the official version and are removed in the build process, see [here](https://github.com/digitoimistodude/air-light/blob/e6ac0a020ae2e83b6c26564929bfdd4d6c4bf1ca/bin/air-move-out.sh#L47-L52).

This is all documented in the [Wiki](https://github.com/digitoimistodude/air-light/wiki/Using-with-Local-by-Flywheel-or-other-3rd-party-development-environment#development-steps-and-custom-post-types-cpts) as well.

Recarding to the official theme version, please leave any feedback items at [trac](https://themes.trac.wordpress.org/ticket/126967) and next time: 1st make sure you have gone through wiki, readme and issues and then open a separate issue if you think there is one. Thank you.
